sections/06-vibrant-bodies.md

```diff
@@ -0,0 +1,201 @@
+
+Echoing this idea of "replaying" and drawing it back to the theme of community, Stephen Voyce imagines a more general mode of what he calls "open source poetics," which is defined by "a decentralized and nonproprietary model of shared cultural codes, networks of dissemination, and collaborative authorship" (407). His view of open communities and their relation to art practice focusses on appropriative, readymade, remix, and "uncreative" strategies, which call into question the role of singular authorship and shift creative emphasis instead towards how poetry contributes to a "poetic commons" of textual references and transformations (408). While, as discussed above, I think it is a mistake to focus only on the citational or appropriative modes of open communities, I nonetheless want to explore Voyce's idea of a commons in which works of art serve, via quotation and systematic transformation, as the "raw materials," or signals, for new works.
+
+Notably, Voyce argues that, underneath the agonistic and oppositional tropes of modernism, open community tactics have been an ongoing feature of avant-garde art movements such as Dada and Fluxus. For Voyce, this method of inter-authorial materiality suggests that avant-garde artists have an ongoing role and responsibility "to create and fortify public domains of open source knowledge, to challenge excessive restrictions placed on language and information, to bring forth marginalized knowledges from a position of inaccessibility to the public at large, and to produce and share artistic tactics and works that challenge intellectual property. That which is at stake is nothing less than open accessibility to culture" (427-8).
+
+## Material Systems: Font Màgica, Chris Welsby, and Tofino
+
+### Font Màgica
+
+For my soundtrack to Izabella Pruska-Oldenhof's film _Font Màgica_, at her request, I used a recording of Chopin's Prelude No. 15 ("Raindrop") as source material, layering and blending it with the sound of radio astronomy events. Since Izabella had initially edited the film to the Chopin piece, I wanted to retain some of the affective logic of Rudolf Serkin's expressive piano performance in my treatment of it. Using a network of amplitude trackers and granular signal processors, the piece's formal structure, its dynamic phrasing, was used as a signal with which to modulate and filter the playback of multiple layers of the Chopin along with the astronomical sounds. The Raindrop prelude's own compositional structure was fed back into the system, refracting itself into multiple layers of sound that are eerily offset from and mis-registered to the original, but which nonetheless carry a strangely warped expressivity. Like Izabella's visuals, it is an overtly refractive piece, full of prismatic colours, distorted ghost images, and passing fragments of a receding original.
```

Comment:
Comment from David: "I don’t think you should refer to Izabella by her first name."